tests/test_wfirst.py

```diff
@@ -541,9 +541,9 @@ def test_wfirst_detectors():
     assert im_2.wcs == im_1.wcs
     assert im_2.dtype == im_1.dtype
     assert im_2.bounds == im_1.bounds
-    np.testing.assert_array_equal(
-        im_2.array, im_1.array,
-        err_msg='Persistence results depend on function used.')
+    #np.testing.assert_array_equal(
+    #    im_2.array, im_1.array,
+    #    err_msg='Persistence results depend on function used.')
```

Comment:
Probably should revert this change and use the linear model above so it matches what happens to im_1.

Then add an additional unit tests that actually check that the fermi persistence model is doing what you think it should be doing.  Right now, the only check is that there are no catastrophic errors when you run the function.  Nothing here checks that the math is right.